code/modules/surgery/tools.dm

```diff
@@ -29,6 +29,7 @@
 		icon_state = "retractor_a"
 
 /obj/item/retractor/advanced/examine(mob/living/user)
+	. = ..()
 	to_chat(user, "<span class = 'notice> It resembles a retractor[tool_behaviour == TOOL_RETRACTOR ? "retractor" : "hemostat"]. </span>")

```

Comment:
is this supposed to resolve to "It resembles a retractorretractor." and "It resembles a retractorhemostat."